scalding-parquet/src/test/java/com/twitter/scalding/parquet/example/TestInputOutputFormat.java

```diff
@@ -79,7 +79,7 @@
   private static final Logger LOG = LoggerFactory.getLogger(TestInputOutputFormat.class);
 
   final Path parquetPath = new Path("target/test/example/TestInputOutputFormat/parquet");
-  final Path inputPath = new Path("src/test/java/org/apache/parquet/hadoop/example/TestInputOutputFormat.java");
```

Comment:
Why this change?

Reply:
This test was copied by @tlazaro in previous PR, and it relies on reading itself to run properly. Since path of copied file is different it started to fail, so I changed it.